.github/workflows/python-publish.yml

```diff
@@ -5,8 +5,28 @@ on:
     types: [published]
     
 jobs:
+  pytest:
+    name: Run pytest
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10.13"
+      - name: Install dependencies
+        run: pip install pytest pytest-md pytest-workflow biopython==1.81 pandas==2.0.2 numpy==1.24.3
```

Comment:
The testing dependencies should be stored in the pyproject.toml file -> https://github.com/EBI-Metagenomics/emgapi/blob/master/pyproject.toml#L68

Reply:
will the github action know to look into that part of the pyproject.toml file for these? or do I need to add anything else to it?

Reply:
ah from looking at the test.yml in that repo I'm assuming it's this line?
`pip install install .[tests]`